backend/kesaseteli/applications/api/v1/serializers.py

```diff
@@ -50,3 +52,13 @@ def update(self, instance, validated_data):
             update_summer_vouchers_using_api_data(summer_vouchers_data, instance)
 
         return super().update(instance, validated_data)
+
+    @transaction.atomic
+    def create(self, validated_data):
+        user = self.context["request"].user
+        company = get_or_create_company_from_eauth_profile(
+            user.oidc_profile.eauthorization_profile
```

Comment:
I guess user.oidc_profile would always be set (required by db schema?) No other kinds of users?

Reply:
It would always be set because it is required by the `EAuthRestAuthentication` here: https://github.com/City-of-Helsinki/yjdh/blob/develop/backend/shared/shared/oidc/auth.py#L123. There should not be any other kinds of users at least for now.